kivy/uix/image.py

```diff
@@ -158,10 +169,49 @@ def get_image_ratio(self):
 
     .. versionadded:: 1.0.8
 
+    .. deprecated:: 2.2.0
+        :attr:`keep_ratio` have been deprecated. Please use `fit_mode`
```

Comment:
We're still suggesting the usage of `keep_ratio`, even if is now deprecated:

https://github.com/kivy/kivy/blob/e2c8f0753337cd693ae5b34de0b3bd3a481d725c/kivy/uix/image.py#L35
https://github.com/kivy/kivy/blob/e2c8f0753337cd693ae5b34de0b3bd3a481d725c/kivy/uix/image.py#L83